src/panel/poi_bloc/images_panel.js

```diff
@@ -0,0 +1,18 @@
+import ImagesPanelView from '../../views/poi_bloc/images.dot'
+import nextTransitionPartial from '../../views/poi_bloc/hour_next_transition_partial.dot'
+import Panel from "../../libs/panel";
+import OsmSchedule from '../../../src/adapters/osm_schedule'
+import Telemetry from "../../libs/telemetry";
+
+function ImagesPanel(block, poi) {
```

Comment:
please use the new syntax
export class ImagePanel
then add a constructor

Reply:
which new syntax?  ES6?